source/elements/oneDAL/source/common_interface/header_files_and_namespaces.rst

```diff
@@ -1,3 +1,61 @@
-===========================
-Header files and namespaces
-===========================
+============
+Header files
+============
+
+oneDAL public identifiers are represented in the following header files:
+
+.. list-table::
+   :widths:  10 50
+   :header-rows: 1
+
+   * - header file
+     - description
+   * - ``onedal/onedal.hpp```
+     - Main header file of oneDAL library.
+   * - ``onedal/algo/%ALGORITHM%/%ALGORITM%.hpp``
+     - An algorithm header file for a particular algorithm.
+       The string %ALGORITHM% should be substituted with the name of the algorithm, for example, ``kmeans`` or ``knn``.
+
+   * - ``onedal/algo/misc/%FUNCTIONALITY%/%FUNCTIONALITY%.hpp``
+     -  A header file for miscellaneous data types and functionality intended to be used by oneDAL algorithms and applications for algorithm customization and optimization on various stages
```

Comment:
this sentence is too long and hard-to-read, but I don't see how to break it in two. what about leaving only this information?

```suggestion
     -  A header file for miscellaneous data types and functionality that is intended to be used by oneDAL algorithms and applications.
```